content/en/users/cloud-compute/auth/_index.md

```diff
@@ -74,7 +83,42 @@ approved you will be able to interact with the infrastructure.
 
 Pre-existing VOs of EGI can be also used on IaaS cloud providers.
 Consult with your VO manager or browse the existing VOs at the [EGI
-Operations Portal](https://operations-portal.egi.eu/vo/).
+Operations Portal](https://operations-portal.egi.eu/vo/). Joining these
+VOs may require a personal X.509 certificate. For getting this certificate,
```

Comment:
This info is now in Check-in VOMS (../check-in/voms)

Reply:
Do we want to remove the part from:
`For getting this certificate` and just link to:
`../voms/#get-a-certificate` ?